web/pgadmin/browser/static/js/MainMenuFactory.js

```diff
@@ -34,10 +49,12 @@ export default class MainMenuFactory {
           MainMenuFactory.electronCallbacks[smName] = sm.callback;
           return {
             ...sm.serialize(),
+            accelerator: MainMenuFactory.convertShortcutToAccelerator(sm.shortcut),
```

Comment:
accelerator will actually register a shortcut. It is not just for labelling. Did you test triggering the shortcut?
Although, it will trigger double shortcut.